__e2e__/root.test.ts

```diff
@@ -23,14 +27,11 @@ beforeAll(() => {
   writeFiles(cwd, {});
 
   // Initialise React Native project
-  runCLI(cwd, ['init', 'TestProject', '--install-pods']);
+  runCLI(cwd, ['init', 'TestProject']);
```

Comment:
Here we don't need to install pods, since this test check only android related things.